PyInstaller/depend/bindepend.py

```diff
@@ -866,6 +866,35 @@ def get_python_library_path():
         if hasattr(sys, 'dllhandle'):
             import _winapi
             return _winapi.GetModuleFileName(sys.dllhandle)
+        elif compat.is_pypy:
+            # PyPy on Windows may not expose sys.dllhandle; search for libpypy*.dll in standard locations.
+            py_major, py_minor = sys.version_info[:2]
+            py_suffix = "t" if compat.is_nogil else ""
+            pypy_dll_names = [
+                f'libpypy{py_major}.{py_minor}-c.dll',
+                f'libpypy{py_major}.{py_minor}{py_suffix}-c.dll',
+                f'libpypy{py_major}{py_minor}-c.dll',
+                f'libpypy{py_major}-c.dll',
+            ]
+            # Search in sys.base_prefix and lib subdirectory
+            # Note: _find_lib_in_libdirs is defined later in this function, but we can use it here
+            # since we're inside the function scope.
```

Comment:
Perhaps this is the case in some parallel universe, but not in this particular one. Trying to  use `_find_lib_in_libdirs` here results in:

```
UnboundLocalError: local variable '_find_lib_in_libdirs' referenced before assignment
```

But this is masked by the fact that `sys.dllhandle` is available and this codepath is not taken.

Also, PyPy does not have a no-GIL variant in the first place.